src/public/app.js

```diff
@@ -0,0 +1,11 @@
+(function() {
+	let failureCount = 0;
+	const maxFailures = 5;
+	document.addEventListener("htmx:sseError", function(evt) {
+		failureCount++;
+		if (failureCount >= maxFailures) {
+			console.log("Max failures reached. Stopping SSE connection.");
+			evt.target.removeAttribute("sse-connect");
```

Comment:
an alternative or additional action here might be to:

1. confirm the user is not authenticated
2. navigate to /login

very optional